src/components/EditProfile/index.js

```diff
@@ -34,7 +34,8 @@ const EditProfile = props => {
       newNickname,
       newShowAge,
       newShowLocation,
-      newLocation.value
+      newLocation.value,
```

Comment:
Image update is now in two different functions. Here, `updateProfilePicture` already updates the pic, but it is also now added to the handleEdit ready function.

Reply:
True. This was a bit confusing for me but it should be clearer now.

Reply:
Seems clear now! :)